frontend/src/components/Item/Metadata/ImageEditor.vue

```diff
@@ -7,8 +7,7 @@
       xl="1"
       lg="3"
       md="4"
-      sm="6"
-      xs="12">
+      sm="6">
```

Comment:
```suggestion
      sm="6"
      cols="12">
```

This is what the xs prop is really named